packages/logger/src/formatter/LogFormatter.ts

```diff
@@ -131,6 +131,13 @@ abstract class LogFormatter {
       }
     }
 
+    if (
```

Comment:
Can we move this to L119 (similar to what we do for `cause`) so we avoid a reassignment?